From e305f541c34d109a0c415048f5779179d802e0f4 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 18 Oct 2022 16:36:47 +0100
Subject: [PATCH] drm/panel: Rename GEM CMA helpers GEM DMA helpers

As a result of [1], DRM_GEM_CMA_HELPER has been replaced by
DRM_GEM_CMA_HELPER.

[1] 4a83c26a1d87 ("drm/gem: rename GEM CMA helpers to GEM DMA helpers")

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 drivers/gpu/drm/panel/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/gpu/drm/panel/Kconfig
+++ b/drivers/gpu/drm/panel/Kconfig
@@ -178,7 +178,7 @@ config DRM_PANEL_ILITEK_ILI9806E
 	tristate "Ilitek ILI9806E-based panels"
 	depends on OF && SPI
 	select DRM_KMS_HELPER
-	depends on DRM_GEM_CMA_HELPER
+	depends on DRM_GEM_DMA_HELPER
 	depends on BACKLIGHT_CLASS_DEVICE
 	select DRM_MIPI_DBI
 	help
@@ -673,7 +673,7 @@ config DRM_PANEL_TPO_Y17P
 	tristate "TDO Y17P-based panels"
 	depends on OF && SPI
 	select DRM_KMS_HELPER
-	depends on DRM_GEM_CMA_HELPER
+	depends on DRM_GEM_DMA_HELPER
 	depends on BACKLIGHT_CLASS_DEVICE
 	select DRM_MIPI_DBI
 	help
